### ISSUE TITLE
Would you be open to having this game available on flathub.org?

### ISSUE THREAD

**Issue by cpba** [SimonLarsen/mrrescue#18]
Flathub is a project that tries to simplify how users install programs on linux-based systems, it's effectively a software "store" or "repo" that works on any distribution.

I've packaged the game and tested it on my system, so far it's working fine. I want to upload it to make it available for everyone, but I'd like to ask for your consent first.

I'll do all the work myself and you won't need to do anything. However, if you want we'll give you access to the flathub repository on github so that you can control how the game appears to people, publish new versions or even remove it if you so wish.

**Comment by nicolas-sabbatini** [comment]
Look at the [LICENSE.md](https://github.com/SimonLarsen/mrrescue/blob/master/LICENSE), they already gave you permission to do it.

**Comment by cpba** [comment]
Yes, I understand the game is partly MIT licensed. This is why I went ahead and published it late last month: https://flathub.org/apps/details/dk.tangramgames.mrrescue

Still, publishing a piece of software can have ramifications that the original author may not necessarily want, even if it's strictly legal to do so I decided to open this issue as a courtesy. I'd also like to keep it open in case the author wishes to be given ownership of the flathub depot.
